test/ui-test/src/screens/StatusWelcomeScreen.py

```diff
@@ -47,6 +49,22 @@ class SeedPhraseComponents(Enum):
     TWENTY_FOUR_BUTTON: str = "switchTabBar_24_words_Button"
     SEEDS_WORDS_TEXTFIELD_template: str = "onboarding_SeedPhrase_Input_TextField_"
     SUBMIT_BUTTON: str = "seedPhraseView_Submit_Button"
+    
+class MainScreen(Enum):
+    PROFILE_NAV_BAR_BUTTON: str = "mainWindow_ProfileNavBarButton"
+    SETTINGS_BUTTON = "settings_navbar_settings_icon_StatusIcon"
```

Comment:
Here, instead of creating the `MainScreen` enum, you can import the existing one that already contains the settings button (import from `StatusMainScreen`, `MainScreenComponents`).